checker/src/main/java/org/checkerframework/checker/initialization/InitializationChecker.java

```diff
@@ -112,6 +112,8 @@ public NavigableSet<String> getSuppressWarningsPrefixes() {
         // The default prefix "initialization" must be added manually because this checker class
         // is abstract and its subclasses are not named "InitializationChecker".
         result.add("initialization");
+        // The prefix to suppress both nullness and initialization warnings.
```

Comment:
The Initialization Checker is independent of the Nullness Checker and shouldn't be tied to it in any way.
Should this be done in the NullnessChecker subclass of this class?